addons/escoria-core/game/esc_inputs_manager.gd

```diff
@@ -339,6 +339,17 @@ func _on_mouse_exited_item(item: ESCItem) -> void:
 # - event: The input event from the click
 func _on_mouse_left_clicked_item(item: ESCItem, event: InputEvent) -> void:
 	if input_mode == INPUT_ALL:
+		if not escoria.action_manager.is_object_actionable(item.global_id):
+			escoria.logger.debug(
+				self,
+				"Ignoring left click on %s with event %s." % [item.global_id, event]
+			)
+
+			# Treat this as a background click now
+			_on_left_click_on_bg(item.global_position)
```

Comment:
You can very often have scenes where the item is located at (0,0), but with an offset to be located at the right place in the room. This could lead the player to walk to a wrong destination.
Therefore I'd perform the click in the exact mouse position at the moment of the input (ie. `event.position`).

Reply:
Good catch. Will fix.